.github/workflows/ci.yaml

```diff
@@ -0,0 +1,135 @@
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 0
+
+      # Required by helm/chart-testing-action
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      # Required by helm/chart-testing-action
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.1.0
+
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        run: |
+          changed=$(ct list-changed)
+          echo "$changed"
+          if [[ -n "$changed" ]]; then
+            echo "::set-output name=changed::true"
+          else
+            echo "::set-output name=changed::false"
+          fi
+
+      - name: Run chart-testing (lint)
+        run: ct lint --lint-conf .github/ct.yaml
+
+  lint-docs:
+    runs-on: ubuntu-latest
+    needs:
+      - lint-chart
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 0
+
+      - name: Run helm-docs
+        run: .github/helm-docs.sh
+
+  kubeval-chart:
+    runs-on: ubuntu-latest
+    needs:
+      - lint-chart
+    strategy:
+      matrix:
+        k8s:
+          # Possible values based on folder names from
+          # https://github.com/instrumenta/kubernetes-json-schema
+          - v1.12.10
+          - v1.13.11
+          - v1.14.10
+          - v1.15.7
+          - v1.16.4
+          - v1.17.4
+          - v1.18.1
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 0
+
+      - name: Run kubeval
+        run: .github/kubeval.sh ${{ matrix.k8s }}
+
+  install-chart:
+    if: needs.lint-chart.outputs.list-changed == 'true'
+    runs-on: ubuntu-latest
+    needs:
+      - lint-chart
+      - kubeval-chart
+    strategy:
+      matrix:
+        k8s:
+          # Possible values based on tag names from:
+          # https://github.com/kubernetes-sigs/kind/releases
+          # Which refers to images from:
+          # https://hub.docker.com/r/kindest/node/tags?page=1&ordering=last_updated
+          - v1.14.10
+          - v1.15.12
+          - v1.16.15
+          - v1.17.17
+          - v1.18.19
+          - v1.19.11
+          - v1.20.7
+          - v1.21.1
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 0
+
+      - name: Create KIND ${{ matrix.k8s }} cluster
+        uses: helm/kind-action@v1.1.0
+        with:
+          node_image: kindest/node:${{ matrix.k8s }}
+          # Possible values based on version tags of kind
+          # https://github.com/kubernetes-sigs/kind/tags
+          version: v0.11.0
+
+      # Required by helm/chart-testing-action
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      # Required by helm/chart-testing-action
+      - uses: actions/setup-python@v2
```

Comment:
(question) 

Is this syntax correct or does `uses` need to be under a `-name´?

If I were to guess this syntax means that all steps after this one needs python?

Reply:
And the order here doesn't really matter much. It's YAML syntax for a map

```yaml
# These three are semantically identical
      - name: Set up chart-testing
        uses: helm/chart-testing-action@v2.1.0

      - uses: helm/chart-testing-action@v2.1.0
        name: Set up chart-testing

      - {
          uses: helm/chart-testing-action@v2.1.0,
          name: Set up chart-testing
        }
```

Reply:
Thanks, I understand I think :)

Just to be sure though, each step is run serially, right?

Reply:
That's right! But each matrix stage is run in parallel.

Kind of the other way around in comparison to Wharf.